### DISCUSSION TITLE
want to use loadVideo but did not want to autoplay , also did not want to use cuevideo

### DISCUSSION THREAD

**Discussion by rakeshrajput537** [PierfrancescoSoffritti/android-youtube-player#1030] [Q&A]
If i use cueVideo it's take time to buffer the video so i want to use loadVideo but in case of loadvideo it's playing the video automatically which i did not want .
I want to play manually but want to finish buffering in advance.

**Comment by PierfrancescoSoffritti** [comment]
You could try pausing the video as soon as it starts playing.

**Reply by rakeshrajput537** [reply]
If i do this then my ExoPlayer video get paused which is currently playing. 
Please let me know either way to do this

**Comment by PierfrancescoSoffritti** [comment]
Yes, only one video at a time can play. It seems like using `cueVideo` is the way to go.

**Comment by rakeshrajput537** [comment]
Hi PierfrancescoSoffritti/android-youtube-player,
We have a cache where we want to pre cache video with buffering state.
Please help me with this.
I am also ready to pay for this feature.
Thanks

**Reply by PierfrancescoSoffritti** [reply]
The videos are managed by the IFrame API. So I don't think this will be possible without downloading the videos in separate ways.
